### ISSUE TITLE
Only see cookies of first window in FF browser then several windows opened

### ISSUE THREAD

**Issue by bapohka** [fcapano/Edit-This-Cookie#206]
Then i use FF i often open new windows.
And then i need to delete cookies from 2+ windows i can't, because i see cookies from active tab of the first window

**Comment by Workshop2** [comment]
Also having this issue.

**Comment by brillb** [comment]
Agreed, it now only shows the cookies from the first window opened in Firefox.  FF 63.0
